src/index.ts

```diff
@@ -43,13 +53,13 @@ class Paragraph {
    * @param {object} params.api - editor.js api
    * @param {boolean} readOnly - read only mode flag
    */
-  constructor({data, config, api, readOnly}) {
+  constructor({ data, config, api, readOnly }: { data: any; config: any; api: any; readOnly: any }) {
```

Comment:
The `data` is not an `any`. You know its structure: 

```ts
interface ParagraphBlockData extends BlockToolData {
  /**
   * Paragraph text content
   */
  text: string;
}
```

Also, we have a type for the `api`. And the `readOnly` is a `boolean`;